src/dotnet/App.Maui/Platforms/Android/AndroidNotificationPermissions.cs

```diff
@@ -0,0 +1,54 @@
+using ActualChat.Notification.UI.Blazor;
+using Android;
+using Android.App;
+using Android.Content.PM;
+using AndroidX.Core.Content;
+
+namespace ActualChat.App.Maui;
+
+public class AndroidNotificationPermissions : INotificationPermissions
+{
+    public AndroidNotificationPermissions()
+    { }
+
+    public Task<PermissionState> GetNotificationPermissionState(CancellationToken cancellationToken)
+    {
+        var activity = Platform.CurrentActivity!;
```

Comment:
there was already generic maui wrapper for permissions:
https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/appmodel/permissions?view=net-maui-7.0&tabs=ios